.gitignore

```diff
@@ -19,6 +19,11 @@ dlp_key.json
 *.avro
 *.avsc
 
+# Don't ignore dataset under solution-test
+!.github/mock-artifacts/*.csv
```

Comment:
is "mock-data" a better name for the folder?

Reply:
Updated.